lib/resources/auditd_rules.rb

```diff
@@ -15,6 +15,7 @@ def initialize(content)
         assignment_regex: /^\s*([^:]*?)\s*:\s*(.*?)\s*$/,
         multiple_values: true,
       }
+      @legacy = nil
```

Comment:
It looks like this class doesn't use `@legacy`. I see it in AuditDaemonRules below, but not in here.